derive/src/pyclass.rs

```diff
@@ -174,7 +216,9 @@ pub(crate) fn impl_pystruct_sequence(
         )
     };
     let fake_ident = Ident::new("pystruct_sequence", struc.span());
-    let class_meta = ClassItemMeta::from_nested(struc.ident.clone(), fake_ident, attr.into_iter())?;
+    let meta = StructSeqItemMeta::from_nested(struc.ident.clone(), fake_ident, attr.into_iter())?;
+    let with_pyimpl = meta.inner()._bool("with_pyimpl")?;
+    let class_meta = ClassItemMeta(meta.0);
```

Comment:
Creating class_meta to share its method looks tricky. How about moving the methods into `impl ClassMetaInner` and use only `StructSeqItemMeta` here?